### PR TITLE
paramterized yaml run from CLI

### PR DESCRIPTION
**Pull request by GargiChakraverty-yb**

### CODE REVIEW

**Review by rakshitjain13 — APPROVED** [review]

File: src/main/java/com/oltpbenchmark/DBWorkload.java [hunk 1]
```diff
@@ -85,6 +88,7 @@ public static void main(String[] args) throws Exception {
         }
 
 
+
```
Comment (shaharuk-yb) [inline src/main/java/com/oltpbenchmark/DBWorkload.java:91]: can you remove this blank line

File: src/main/java/com/oltpbenchmark/DBWorkload.java [hunk 2]
```diff
@@ -112,7 +116,16 @@ public static void main(String[] args) throws Exception {
...
+            {
+                String[] params=null;
+                if (argsLine.hasOption("params")) {
+                    params = argsLine.getOptionValues("params");
+                    LOG.info("Creating modified temporary input yaml with passed parameters from : "+ configFile);
+                    configFile = replaceParametersInYaml(params,configFile);
+                }
                 xmlConfig = buildConfigurationFromYaml(configFile);
+            }
+
```
Comment (shaharuk-yb) [inline src/main/java/com/oltpbenchmark/DBWorkload.java:128]: remove the blank line

File: src/main/java/com/oltpbenchmark/DBWorkload.java [hunk 3]
```diff
@@ -968,4 +979,47 @@ private static boolean isBooleanOptionSet(CommandLine argsLine, String key) {
...
+            return newPath;
+
+        } else {
+            throw new IllegalArgumentException(renderResult.getErrors().stream()
+                .map(Object::toString)
+                .collect(Collectors.joining(",")));
+        }
+    }
 }
+
```
Comment (shaharuk-yb) [inline src/main/java/com/oltpbenchmark/DBWorkload.java:1024]: only one blank line is expected towards the end of the file

**Review by shaharuk-yb — CHANGES_REQUESTED** [review]
please make the requested changes

**Review by shaharuk-yb — APPROVED** [review]
LGTM

**Pull request merged**